doc-content/enterprise-only/openshift/hacep-client-proc.adoc

```diff
@@ -9,6 +9,17 @@ You must adapt your CEP client code to communicate with the HA CEP server image.
 . Extract the contents of the file and then uncompress the `{PRODUCT_FILE}-openshift-drools-hacep-distribution.zip` file.
 . Change to the `openshift-drools-hacep-distribution/sources` directory.
 . Review and modify the client code based on the sample project in the `sample-hacep-project/sample-hacep-project-client` directory. Ensure that the code fulfills the additional requirements described in <<hacep-clientcode-ref>>.
+. If you want to update the KJAR version in an implementation that uses the method described in <<hacep-server-maven-proc>>, add a `UpdateKJarGAV` call to the client, similar to the following code:
++
+[source,java]
+----
+        TopicsConfig envConfig = TopicsConfig.getDefaultTopicsConfig();
+        Properties props = getProperties();
+        try (RemoteStreamingKieSession producer = RemoteStreamingKieSession.create(props, envConfig)){
+            producer.updateKJarGAV("org.kie:fake-jar:0.1-SNAPSHOT");
```

Comment:
I know it is a sample but please do not use a -SNAPSHOT GAV, it works but we should not promote it. Just change it like `org.kie:fake-jar:0.1`